os/common/ext/SONiX/SN32F2xx/system_SN32F240.c

```diff
@@ -22,11 +22,10 @@
  *
  ******************************************************************************/
 
-
 #include <stdint.h>
 #include <system_SN32F2xx.h>
 #include <mcuconf.h>
-
+#include <sn32_sys1.h>
```

Comment:
This one can be probably moved to `os/hal/ports/SN32/SN32F240/hal_lld.h` safely. Other than that, PR is good. Please consider pushing upstream